crates/turborepo-lib/Cargo.toml

```diff
@@ -42,7 +42,7 @@ reqwest = { workspace = true, default_features = false, features = ["json"] }
 rustc_version_runtime = "0.2.1"
 semver = { workspace = true }
 serde = { workspace = true, features = ["derive"] }
-serde_json = { workspace = true }
```

Comment:
- why don't we need to keep the `worksapce = true` bit?
- does adding the `turboVersion` require reading comments (i.e. does this soft fork need to be in this PR or can we evaluate separately for reading comments)?

Reply:
1. `workspace = true` tells cargo to inherit the version from the root `cargo.toml`. I'd have to make this change at the root in order to get that sorted.
2. This is the first time that we're actually reading `turbo.json` in Rust so that's why this change is included in this PR.